pkg/settings/setting.go

```diff
@@ -23,36 +24,38 @@ var (
 	provider       Provider
 	InjectDefaults string
 
-	AgentImage                          = NewSetting("agent-image", "rancher/rancher-agent:v2.6-head")
-	AgentRolloutTimeout                 = NewSetting("agent-rollout-timeout", "300s")
-	AgentRolloutWait                    = NewSetting("agent-rollout-wait", "true")
-	AuthImage                           = NewSetting("auth-image", v32.ToolsSystemImages.AuthSystemImages.KubeAPIAuth)
-	AuthTokenMaxTTLMinutes              = NewSetting("auth-token-max-ttl-minutes", "0") // never expire
-	AuthorizationCacheTTLSeconds        = NewSetting("authorization-cache-ttl-seconds", "10")
-	AuthorizationDenyCacheTTLSeconds    = NewSetting("authorization-deny-cache-ttl-seconds", "10")
-	AzureGroupCacheSize                 = NewSetting("azure-group-cache-size", "10000")
-	CACerts                             = NewSetting("cacerts", "")
-	CLIURLDarwin                        = NewSetting("cli-url-darwin", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-darwin-amd64-v1.0.0-alpha8.tar.gz")
-	CLIURLLinux                         = NewSetting("cli-url-linux", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-linux-amd64-v1.0.0-alpha8.tar.gz")
-	CLIURLWindows                       = NewSetting("cli-url-windows", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-windows-386-v1.0.0-alpha8.zip")
-	ClusterControllerStartCount         = NewSetting("cluster-controller-start-count", "50")
-	EngineInstallURL                    = NewSetting("engine-install-url", "https://releases.rancher.com/install-docker/20.10.sh")
-	EngineISOURL                        = NewSetting("engine-iso-url", "https://releases.rancher.com/os/latest/rancheros-vmware.iso")
-	EngineNewestVersion                 = NewSetting("engine-newest-version", "v17.12.0")
-	EngineSupportedRange                = NewSetting("engine-supported-range", "~v1.11.2 || ~v1.12.0 || ~v1.13.0 || ~v17.03.0 || ~v17.06.0 || ~v17.09.0 || ~v18.06.0 || ~v18.09.0 || ~v19.03.0 || ~v20.10.0 ")
-	FirstLogin                          = NewSetting("first-login", "true")
-	GlobalRegistryEnabled               = NewSetting("global-registry-enabled", "false")
-	GithubProxyAPIURL                   = NewSetting("github-proxy-api-url", "https://api.github.com")
-	HelmVersion                         = NewSetting("helm-version", "dev")
-	HelmMaxHistory                      = NewSetting("helm-max-history", "10")
-	IngressIPDomain                     = NewSetting("ingress-ip-domain", "sslip.io")
-	InstallUUID                         = NewSetting("install-uuid", "")
-	InternalServerURL                   = NewSetting("internal-server-url", "")
-	InternalCACerts                     = NewSetting("internal-cacerts", "")
-	IsRKE                               = NewSetting("is-rke", "")
-	JailerTimeout                       = NewSetting("jailer-timeout", "60")
-	KubeconfigGenerateToken             = NewSetting("kubeconfig-generate-token", "true")
```

Comment:
Can you avoid this big reformat on above 20 lines, makes git-blame harder in the future.

Reply:
I originally did not reformat those lines, but CI failed because it was not formatted correctly. I can move the settings with comments to the end as you suggested in your previous comment, which should fix this.